### PR TITLE
[PT-BR] translation of transitions

### PR DESCRIPTION
**Pull request by FelipeAlves99**
Hey guys,

This PR translates the file **transitions.md** to portuguese.
The folder **Community** was created at pt-br as well.

@glaucia86 @khaosdoctor @nodejs/nodejs-pt

### CODE REVIEW

File: locale/pt-br/blog/community/transitions.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+que irá administrar o futuro do Node.js e abri-lo para a
+comunidade de uma maneira que não estava disponível antes. Organizações como
+IBM, SAP, Apigee, F5, Fidelity, Microsoft, PayPal, Red Hat, e outras estão
+patrocinando a Fundação, e eles estão adicionando mais contribuidores ao projeto.
+A missão da Fundação é acelerar a adoção de Node e garantir
+que o projeto seja conduzido pela comunidade sob uma governança transparente e de modelo
+aberto.
+
+Sob a proteção da Fundação, o projeto Node.js está entrando para a próxima
+fase de maturidade e adotando um modelo em que não há nenhum BD ou líder de projeto.
```
Comment (khaosdoctor) [inline locale/pt-br/blog/community/transitions.md]: This sentence is weird... It does not make sense

Comment (khaosdoctor) [reply locale/pt-br/blog/community/transitions.md]: Can you please post the original one so we can find a better translation?

File: locale/pt-br/blog/community/transitions.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Em vez disso, a direção técnica do projeto será estabelecida por um
+comitê técnico de direção executado com um modelo de gorvernança aberto. Houve muita
```
Comment (khaosdoctor) [inline locale/pt-br/blog/community/transitions.md]:
```suggestion
comitê técnico de direção executado com um modelo de governança aberto. Houve muita
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

File: locale/pt-br/blog/community/transitions.md (hunk 1 above)
Comment (FelipeAlves99) [reply locale/pt-br/blog/community/transitions.md]: Sure,

Under the aegis of the Foundation, the Node.js project is entering the next
phase of maturity and adopting a model in which there is no BD or project lead.

aegis means protection.
I think BD is Business Director, but i'm not sure about it, so I didn't change it.

File: locale/pt-br/blog/community/transitions.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+(https://www.joyent.com/blog/introducing-the-nodejs-foundation),
+que irá administrar o futuro do Node.js e abri-lo para a
+comunidade de uma maneira que não estava disponível antes. Organizações como
+IBM, SAP, Apigee, F5, Fidelity, Microsoft, PayPal, Red Hat, e outras estão
+patrocinando a Fundação, e eles estão adicionando mais contribuidores ao projeto.
+A missão da Fundação é acelerar a adoção de Node e garantir
+que o projeto seja conduzido pela comunidade sob uma governança transparente e de modelo
+aberto.
+
+Sob a proteção da Fundação, o projeto Node.js está entrando para a próxima
```
Comment (khaosdoctor) [inline locale/pt-br/blog/community/transitions.md]:
```suggestion
Sob a proteção da Fundação, o Node.js está entrando em sua próxima
```

File: locale/pt-br/blog/community/transitions.md (hunk 1 above)
Comment (khaosdoctor) [inline locale/pt-br/blog/community/transitions.md]:
```suggestion
fase de maturidade, adotando um modelo onde não há nenhum BD ou líder de projeto.
```

Comment (khaosdoctor) [reply locale/pt-br/blog/community/transitions.md]: Did some changes, I think that's more like it. I really do not know how to translate BD to Portuguese too, so let's leave as it is

**Review by khaosdoctor — CHANGES_REQUESTED** [review]

**Comment by FelipeAlves99** [comment]
Hey @khaosdoctor, just made the corrections. 
Hope its in a good and understandable state now.

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
Awesome! LGTM @Trott & @MaledongGit

**Pull request merged**